frontend/src/metabase-lib/v1/expressions/pratt/lexifier.unit.spec.ts

```diff
@@ -537,21 +537,6 @@ describe("lexify", () => {
       );
     });
 
-    it("handles brackets that aren't properly and have whitespace between them", () => {
-      const { tokens } = lexify("foo  ]");
-      expect(tokens).toEqual(
-        [
-          {
-            type: FIELD,
-            pos: 0,
-            text: "foo  ]",
-            value: "foo",
-          },
-          { type: END_OF_INPUT, pos: 6, text: "\n" },
-        ].map(asToken),
-      );
-    });
```

Comment:
This is no longer supported, but just returns another error message.

Reply:
The error message says `Invalid character: ]` but `]` is a valid character.

Same error when expression is `]`.